<launch>

  <arg name="uav_name" default="$(optenv UAV_NAME uav)"/>
  <arg name="profiler" default="$(optenv PROFILER false)" />

  <group ns="$(arg uav_name)">

    <node pkg="nodelet" type="nodelet" name="rtk_republisher" args="load mrs_odometry/RtkRepublisher $(arg uav_name)_nodelet_manager" output="screen">

      <rosparam file="$(find mrs_odometry)/config/rtk_republisher.yaml" />
      <param name="enable_profiler" value="$(arg profiler)" />

      <!-- Subscribers -->
      <remap from="~odom_in" to="ground_truth" />
      <remap from="~rtk_out" to="rtk_gps/global" />

      <!-- Publishers -->
      <remap from="~profiler" to="profiler" />

    </node>

  </group>

</launch>
